token-metadata/program/README.md

```diff
@@ -0,0 +1,53 @@
+# Metadata
```

Comment:
Hey I'll remove this MD file but keep the readme so that the Github Contents table feature still works.

![image](https://user-images.githubusercontent.com/315499/113341801-3d6a0a80-92f3-11eb-8d51-e369134804ae.png)